hasura/graphql-engine
# fix scrollbar on table rows - close #6158

### Description


Scrollbar on bottom is always visible even when there's rows overflowing to the bottom of page.
![Captura de tela de 2021-03-30 16-09-11](https://user-images.githubusercontent.com/4965156/113043051-4da6ac00-9172-11eb-9dbd-2761fd64aa63.png)

### Changelog


### Affected components



### Related Issues



#6158 

### Solution and Design



### Steps to test and verify


Access a table with enough data to make overflow both vertical and horizontal, and check to see if bottom scroll appears.

### Limitations, known bugs & workarounds



### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
